exercises/concept/stage-heralding/.docs/instructions.md

```diff
@@ -0,0 +1,108 @@
+
+!!! note
+    This exercise is **not** about learning regular expressions.
+    It's about learning to use regex within Julia.
+    The hints contain a regular expression that you can use if you don't want to write one yourself.
+
+The list you're given contains the following information for each talk:
+
+- its title,
+- the speaker's name,
+- their pronouns (optional),
+- their organisation (optional),
+- the start time of the talk,
+- the start of the Q&A session (optional),
+- and the end time of the talk.
+
+A full entry has the following format:
+
+```text
+»Speedrunning 101« – Sasha Duda Krall (they/them), from GDQU. Start: 13:00, Q&A: 13:20, End: 13:30
+```
+
+Here, the title is `Speedrunning 101`, the speaker is called `Sasha Duda Krall`, from `GDQU`, and uses `they/them` pronouns.
+The talk starts at `13:00`, there will be a Q&A starting at `13:20`, and the entire session ends at `13:30`.
+
+!!! note
+    Since names may contain regular dashes `-`, an en dash `–` is used to separate the speaker and talk name.
+
+The moderation card that your program should generate for that talk looks like this:
+
+```text
+- Our next speaker is Sasha Duda Krall, from GDQU
+- Their talk is called »Speedrunning 101«
+- They will answer your questions in the Q&A session at the end of the talk, starting at 13:20
+
+13:00 - 13:20 - 13:30
+```
+
+The entire list may look like this:
+
+```text
+»Speedrunning 101« – Sasha Duda Krall (they/them), from GDQU. Start: 13:00, Q&A: 13:20, End: 13:30
+»How I learned to say Farewell« – Madeline (she/her). Start: 13:40, Q&A: 14:00, End: 14:05
+»How To Find a Good Title For Your Conference Talk« – Vítězslav Kruse (he/him). Start: 14:15, End: 14:40
+»Why Emoji Matter❣« – Ash van Amelsvoort, from the University of 🧬🧪⚛. Start: 14:50, Q&A: 15:05, End: 15:10
+```
+
+The corresponding cards are:
+
+```text
+- Our next speaker is Madeline
+- Her talk is called »How I learned to say Farewell«
+- She will answer your questions in the Q&A session at the end of the talk, starting at 14:00
+
+13:40 - 14:00 - 14:05
+```
+
+```text
+- Our next speaker is Vítězslav Kruse
+- His talk is called »How To Find a Good Title For Your Conference Talk«
+- There will not be a Q&A session.
+
+14:15 - NO Q&A - 14:40
+```
+
+```text
+- Our next speaker is Ash van Amelsvoort, from the University of 🧬🧪⚛
+- Ash's talk is called »Why Emoji Matter❣«
+- Ash will answer your questions in the Q&A session at the end of the talk, starting at 15:05
+
+14:50 - 15:05 - 15:10
+```
+
+A few things to note here:
+
+- If no organisation is given for a speaker, leave it out in the moderation card.
+- If no time for the Q&A session is given, the speaker opted against answering questions.
+- If the speaker does not specify pronouns, use the speaker's first name instead.
+
+## 1. Write a regex that captures the information you need to generate the cards
```

Comment:
Maybe just do:

```suggestion
## 1. Capture the information needed to generate the cards
```

Per https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#file-docshintsmd:

> Task headings should describe the what of the task, not the how.

Reply:
Probably want to specify that we want a regex, otherwise people will use other tools.

Reply:
But that would be explained in the task's text, right?